example/src/main/kotlin/com/expedia/graphql/sample/GraphQLRequest.kt

```diff
@@ -1,7 +1,18 @@
 package com.expedia.graphql.sample
 
+import com.expedia.graphql.sample.context.MyGraphQLContext
+import graphql.ExecutionInput
+
 data class GraphQLRequest(
     val query: String,
     val operationName: String? = null,
     val variables: Map<String, Any>? = null
 )
+
+fun GraphQLRequest.getExecutionInput(graphQLContext: MyGraphQLContext? = null): ExecutionInput =
```

Comment:
since you are converting it to execution input maybe rename it to `to` instead of `get`?

Reply:
👍 I had it as an external method and then moved it. I will rename